### PR TITLE
Add show news, events to profile

### PR DESCRIPTION
**Pull request by AbdulRustam**
## JIRA ticket

_Please provide the URL for the JIRA ticket._

## Code reviewers

_Please list your code reviewers here._

## Summary of issue

Show news, events in user profile

## Summary of change

Add news and events in user profile.
Change view in search users, news.

## Review checklist

- [x] Rubocop doesn't raise any warnings and offenses
- [x] New features are covered with new specs
- [x] All specs pass

### CODE REVIEW

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.69%** covered at **3.85** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.69%** covered at **3.85** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.69%** covered at **3.85** hits/line

File: app/controllers/email_change_requests_controller.rb [hunk 1]
```diff
@@ -8,10 +8,12 @@ class EmailChangeRequestsController < ApplicationController
   def update; end
 
   def new
+    @user = current_user
```
Comment (omaik) [inline app/controllers/email_change_requests_controller.rb]: why do you need to do this? `current_user` is accessible in the view

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ class UsersController < ApplicationController
   before_action :authenticate_user!, except: %i[show index]
   has_scope :by_roles, type: :array
   respond_to :js, :html, only: %i[index]
+  before_action :find_user, only: %i[news events show edit_password update_password]
```
Comment (omaik) [inline app/controllers/users_controller.rb]: why is this needed for `edit_password` and `update_password`

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -28,8 +29,14 @@ def update
 
   def edit_password; end
 
-  def show
-    @user = User.find(params[:id])
+  def show; end
+
+  def news
```
Comment (omaik) [inline app/controllers/users_controller.rb]: these 2 methods should go into their own controllers.

File: app/views/users/show.html.haml [hunk 4]
```diff
@@ -2,10 +2,16 @@
...
-            = t ('users.edit.profile')
-        .card-body
-          = render 'form', user: @user
+    .col-md-5.col-xl-3
+      = render 'menu', user: @user
+    .col-md-7.col-xl-9
+      = render 'form', user: @user
+
+
+    / .col-md-8.col-xl-12
```
Comment (omaik) [inline app/views/users/show.html.haml]: commented code

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.11%** covered at **3.82** hits/line
